rust/src/lib/ip.rs

```diff
@@ -961,3 +963,27 @@ fn is_none_or_empty_mptcp_flags(v: &Option<Vec<MptcpAddressFlag>>) -> bool {
 fn default_allow_extra_address() -> bool {
     true
 }
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+#[serde(rename_all = "lowercase")]
+pub enum AddressFamily {
+    IPv4,
+    IPv6,
+    Unknown,
+}
+
+impl From<u8> for AddressFamily {
+    fn from(d: u8) -> Self {
+        match d {
+            AF_INET => AddressFamily::IPv4,
+            AF_INET6 => AddressFamily::IPv6,
+            _ => AddressFamily::Unknown,
+        }
+    }
+}
+
+impl Default for AddressFamily {
+    fn default() -> Self {
+        Self::Unknown
```

Comment:
As you mentioned in commit message, the default should be `IPv4`

Reply:
Done, thanks!